### DISCUSSION TITLE
Deploy pretix with custom changes

### DISCUSSION THREAD

**Discussion by carl-egge** [pretix/pretix#3477] [Q&A]
Hej guys,

I wanted to deploy the pretix application on a server, including some custom changes. I don't want to create a plugin that extends the pretix functionality, but I rather wanted to change some things in the code to fit my personal preferences (like changing the event db model, ...). Afterwards, I want to deploy my custom pretix in production using nginx and gunicorn (similar to the Small Scale Deployment). The documentation holds information on how to create a Dev installation and run the test server, and also on how to deploy to production using docker or the pypi package of pretix. I was wondering what I would need to do and think about if I would like to deploy my version of the pretix code using nginx and gunicorn.

I see for example that the nginx config is referencing the static files from the pretix package in the pip env when following the small scale deployment. When I follow a similar approach but with my custom pretix code, I will not have these static files in the env.

I am not an experienced Django developer and I am happy for any help!
Best regards,
Carl

**Comment by raphaelm** [comment]
> like changing the event db model

Sorry, but with that you're out of the scope I want to give guidance on. Of course you can modify pretix as you like, but it's not intended to be done and I'll not support it :)
